### ISSUE TITLE
Proposal to implement the use of environment variables to not expose the secrets

### ISSUE THREAD

**Issue by Chilace** [Suppen/TediCross#37]
Currently I use a quick workaround for this:
`./lib/settings.js`
```
/**************************************
 * Write it back to the settings file *
 **************************************/

fs.writeFile(settingsPath, JSON.stringify(settings, null, "\t"), (err) => {
	if (err) {
		console.error("Could not save changes to the settings!");
	}
});

// Use of .env
settings.telegram.auth.token = process.env.T_TOKEN || settings.telegram.auth.token;
settings.discord.auth.token = process.env.D_TOKEN || settings.discord.auth.token;

/******************************
 * Export the settings object *
 ******************************/

module.exports = settings;
```

**Comment by Jovaage** [comment]
I think we should implement several ways of providing settings to the app.
Enviroment, arguments and from file. (see [nconf](https://www.npmjs.com/package/nconf) or [config](https://www.npmjs.com/package/config))

**Comment by Suppen** [comment]
This is now available in commit c76d73a0b7f144001f52ba79a485c1b881a060c9. To use it, set `telegram.auth.token` and/or `discord.auth.token` in the settings file to `"env"`. The environment variables are `TELEGRAM_BOT_TOKEN` and `DISCORD_BOT_TOKEN`.

This has been documented in the README file
